boards/riscv/it8xxx2_evb/it8xxx2_evb.dts

```diff
@@ -27,19 +22,6 @@
 		zephyr,code-partition = &slot0_partition;
 		zephyr,keyboard-scan = &kscan0;
 	};
-
-	pwmleds {
-		compatible = "pwm-leds";
-		/* NOTE: &pwm number needs same with channel number */
-		led0: led_0 {
-			pwms = <&pwm7 PWM_CHANNEL_7 PWM_POLARITY_INVERTED>;
-			label = "LED0_GREEN";
-		};
-		led1: led_1 {
-			pwms = <&pwm0 PWM_CHANNEL_0 PWM_POLARITY_NORMAL>;
-			label = "LED1_BLUE";
-		};
-	};
```

Comment:
Does this have anything to do with adding the tach driver? If not can you please make this a different commit?

Reply:
Ok, I will create another commit for cleanup.

Reply:
thanks